README.md

```diff
@@ -39,656 +39,17 @@ PLEASE NOTE that `Azuererm` subcommands are experimental and of alpha quality. N
 
 ## Configuration
 
-### ASM mode
-For this plugin to interact with Azure's REST API, you will need to give Knife
-information about your Azure account and credentials. The easiest way to do
-this is to sign in to the Azure portal and download a publishsettings file
-from https://manage.windowsazure.com/publishsettings/index?client=xplat to a
-local file system location, and
-then refer to the local file via an entry in your knife.rb:
+1. [ASM Mode] (docs/configuration.md#asm-mode)
+2. [ARM Mode] (docs/configuration.md#arm-mode)
```

Comment:
Title here can be `ASM Configuration` and `ARM Configuration` to avoid repetition of text `ASM Mode` and `ARM Mode`